src/receivers/HTTPReceiver.ts

```diff
@@ -70,6 +72,12 @@ export interface HTTPReceiverOptions {
   installerOptions?: HTTPReceiverInstallerOptions;
 }
 
+export interface CustomRoute {
+  path: string;
+  method: string | string[];
+  callback: (req: IncomingMessage, res: ServerResponse) => void;
```

Comment:
Although I think `callback` is fine, I feel that `handler` may be a good option too as each element in `customRoutes` is not a function. When I hear "callback", I tend to imagine the situation where a function accepts another function to run internally & asynchronously.

For reference, Express call this kind of function as "handler" https://expressjs.com/en/starter/basic-routing.html
Koa-router accepts middleware: https://github.com/koajs/router/blob/master/API.md

Reply:
Great point, I agree with you, will change it to `handler`!